project/Dependencies.scala

```diff
@@ -28,15 +28,16 @@ object Dependencies {
   val h2specExe = "h2spec" + DependencyHelpers.exeIfWindows
   val h2specUrl = s"https://github.com/summerwind/h2spec/releases/download/v${h2specVersion}/${h2specName}.zip"
 
-  val scalaTestVersion = "3.1.4"
+  val scalaTestVersion = "3.2.9"
   val specs2Version = "4.10.6"
-  val scalaCheckVersion = "1.14.3"
+  val scalaCheckVersion = "1.15.4"
 
   val scalafixVersion = _root_.scalafix.sbt.BuildInfo.scalafixVersion // grab from plugin
 
   val scala212Version = "2.12.17"
   val scala213Version = "2.13.10"
-  val allScalaVersions = Seq(scala213Version, scala212Version)
+  val scala3Version = "3.2.2"
```

Comment:
will we go ahead and upgrade to Scala 3.3.0?

Reply:
I would prefer to do this in a separate PR.